test/commons/dom/is-focusable.js

```diff
@@ -184,89 +184,221 @@ describe('is-focusable', function() {
       assert.isFalse(axe.commons.dom.isFocusable(el));
     });
 
-    it('should return false for a div with no tabindex', function() {
+    it('should return false for a div with no tabindex', function () {
       fixture.innerHTML = '<div id="target"></div>';
       var el = document.getElementById('target');
       flatTreeSetup(fixture);
 
       assert.isFalse(axe.commons.dom.isFocusable(el));
     });
+    describe('options', function () {
+      describe('programatically', function () {
+        describe('true (default)', function () {
```

Comment:
This is kind of a personal preference, but when it comes to boolean options I prefer defaults to be falsey. It's not intuitive (to me) for undefined and true to behave the same, but for false to be different. Much of axe-core works that way, so it's more consistent doing this too.

I propose using `ifProgrammatic` with a falsey default.